x-pack/plugins/apm/server/saved_objects/apm_indices.ts

```diff
@@ -32,4 +32,9 @@ export const apmIndices: SavedObjectsType = {
       },
     },
   },
+  management: {
+    importableAndExportable: true,
+    icon: 'apmApp',
+    getTitle: () => 'APM Settings - Indices',
```

Comment:
Should this be translated?

Reply:
Good catch, lemme check the other titles